indexFinal.html

```diff
@@ -5,53 +5,49 @@
-                    <textarea id="interest"></textarea>     
-                    <div class="required-field"><span class="asterisk"> * </span> Required field </div>
-                </div>
-
-                <div class="vertical"></div>
+            <div class="outside-wrapper">
 
-                <div class="right">
+                <form class="flex-wrapper" name="message-form">
 
-                    <fieldset>
-                        <legend> Anrede </legend>
-                        <label> <input type="radio" name="Anrede" value="Frau"> Frau </label>
-                        <label> <input type="radio" name="Anrede" value="Herr"> Herr </label>
-                        <label> <input type="radio" name="Anrede" value="Keine-Angabe"> Keine Angabe </label>
-                    </fieldset>
-            
-                    <label for="name">Ihr Name</label>
-                    <input type="text" id="name" name="name"/>
-                    <div class="required-field"><span class="asterisk"> * </span> Required field </div>
-     
-                    <label for="email"> Ihr E-Mail-Adresse </label>
-                    <input type="email" id="email" name="email"/>
-                    <div class="required-field"><span class="asterisk"> * </span> Required field </div>
+                    <div class="left">
+                        
+                        <header>
+                            <h1>Schreiben Sie <br /> uns eine <br /> Nachricht</h1>
+                            <hr />
+                        </header>
 
-                    <div class="send-button-wrapper">
-                        <button type="submit" name="send-button" class="send-button" onclick= "inputValue()"> Send </button>
+                        <label for="message"> Ich interessiere mich für ...</label>
+                        <textarea id="message"></textarea>     
+                        <div class="required-field"><span class="asterisk"> * </span> Required field </div>
                     </div>
-
-                    <script src="scriptFinal.js"></script>
-
-                </div>
-
-            </form> 
+                    <div class="vertical"></div>
+                    <div class="right">
+                        <fieldset>
+                            <legend> Anrede </legend>
+                            <label> <input type="radio" name="Anrede" value="Frau"> Frau </label>
+                            <label> <input type="radio" name="Anrede" value="Herr"> Herr </label>
+                            <label> <input type="radio" name="Anrede" value="Keine-Angabe"> Keine Angabe </label>
+                        </fieldset>
+                    
+                        <label for="name">Ihr Name</label>
+                        <input type="text" id="name" name="name"/>
+                        <div class="required-field"><span class="asterisk"> * </span> Required field </div>
```

Comment:
**suggestion(nitpick)** This is a block that is repeated twice. Consider wrapping `label`, `input` & `required-filed` in a `div` and set margins to the wrapper div instead of having the `label` have margin-top.